cms/djangoapps/contentstore/views/videos.py

```diff
@@ -38,6 +37,7 @@
 )
 from opaque_keys.edx.keys import CourseKey
 from pytz import UTC
+from storages.backends.s3boto import S3BotoStorage, S3Key
```

Comment:
Since the `boto` storage is [now deprecated](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#migrating-boto-to-boto3), could you replace this with `storages.backends.s3boto3.S3Boto3Storage`